configs/discord.json

```diff
@@ -9,7 +9,7 @@
     "lvl3": "[class*='contentWrapperInner'] h3",
     "lvl4": "[class*='contentWrapperInner'] h4",
     "lvl5": "[class*='contentWrapperInner'] h5",
-    "text": "[class*='contentWrapperInner'] p, [class*='contentWrapperInner'] li"
+    "text": "[class*='contentWrapperInner'] p, [class*='contentWrapperInner'] li, [class*='contentWrapperInner'] td"
```

Comment:
Hey @typpo,

I'd recommend to make the selector more specific, otherwise it will also react records for the `TYPE` for example, which might add a lot of noise to the results.

Limiting it to the `first-child` might be safer.